pkgs/applications/file-managers/clifm/default.nix

```diff
@@ -1,22 +1,36 @@
-{ stdenv, lib, fetchFromGitHub, libcap, acl, file, readline, python3 }:
+{
+  stdenv,
```

Comment:
Is that atrocity the end result of RFC? Look how they massacred the set.
My apologies...